mkt/feed/management/commands/generate_feed.py

```diff
@@ -0,0 +1,17 @@
+from django.core.management.base import BaseCommand
+
+from mkt.feed.fakedata import generate_feed_data
+
+
+class Command(BaseCommand):
+    """
+    Usage:
+
+        python manage.py generate_feed
+
+    """
+
+    help = 'Generate example feed data for development/testing'
+
+    def handle(self, *args, **kwargs):
+        generate_feed_data()
```

Comment:
Good question, I was running reindex each time due to the ES problems we discussed earlier. I believe it should index as it goes but I'll check.